server/Dockerfile

```diff
@@ -3,12 +3,12 @@ LABEL maintainer="info@camptocamp.com"
 
 # let's copy a few of the settings from /etc/init.d/apache2
 ENV APACHE_CONFDIR=/etc/apache2 \
-    APACHE_ENVVARS=$APACHE_CONFDIR/envvars
+    APACHE_ENVVARS=/etc/apache2/envvars \
 # and then a few more from $APACHE_CONFDIR/envvars itself
-ENV APACHE_RUN_USER=www-data \
+    APACHE_RUN_USER=www-data \
     APACHE_RUN_GROUP=www-data \
     APACHE_RUN_DIR=/var/run/apache2 \
-    APACHE_PID_FILE=$APACHE_RUN_DIR/apache2.pid \
+    APACHE_PID_FILE=/var/run/apache2/apache2.pid \
```

Comment:
Why all the changes above? 
(except removing ENV :-))

Reply:
Because the `$APACHE_RUN_DIR` was evaluated to an empty string. Looks like you cannot use the ENVs declared in the same statement...

Reply:
OK, thanks :-)